api/src/Repository/AppointmentRepository.php

```diff
@@ -118,6 +118,33 @@ public function getThreeLastAppointmentsRepairersIds(User $user): array
         return array_column($qb->getQuery()->getArrayResult(), 'repairerId');
     }
 
+    public function findOneByCustomerAndUserRepairer(User $customer, User $userRepairer): ?Appointment
```

Comment:
It would be nice to add a test in `api/tests/Appointments/Security/PostTest.php` for the case where `$userRepairer` has more than one repairer